cms/templates/course_outline.html

```diff
@@ -50,6 +51,45 @@ <h2 class="title title-3">${_("This course was created as a re-run. Some manual
     </div>
   </div>
   %endif
+
+  %if ora1_deprecation_info.get('ora1_components') or ora1_deprecation_info.get('feature_enabled'):
```

Comment:
I'm in agreement with @ormsbee here that the code shouldn't have any special ORA1 knowledge. IMO this deprecation info should be a list of features that are being used but which are deprecated. It would be up to the server to generate a good error message, and then this page would show a list of deprecation alerts.

This is another use case where I would like to see a first class 'feature' extension point that could be registered with the platform. Then a feature could declare itself as being deprecated, and could implement a method to determine if a course is using it, and to then generate an appropriate user message.